tawazi/errors.py

```diff
@@ -1,13 +1,13 @@
 """Module for custom errors raised by Tawazi."""
 
 
-class TawaziBaseException(BaseException):
+class TawaziBaseError(Exception):
```

Comment:
maybe just `TawaziError` ? It's simpler imho

Reply:
done